packages/mail/lib/sendMailTemplate.js

```diff
@@ -32,6 +34,39 @@ const getTemplate = (name) => {
   return contents
 }
 
+const envMergeVars = []
+
+if (FRONTEND_BASE_URL) {
+  envMergeVars.push({
+    name: 'frontend_base_url',
+    content: FRONTEND_BASE_URL
+  })
+}
+if (SG_FONT_FACES) {
+  envMergeVars.push({
+    name: 'SG_FONT_FACES',
```

Comment:
More recent email templates use lowercase merge variables. Would adopt here and `SG_FONT_STYLE_...`

(More recent email templates use "Handlebars" as a templating language e.g. `{{sg_font_faces}}` and we aim to replace all MailChimp/Mandrill templating gradually. "Handlebars" allows for more logic inside templates.)